data/categories.txt

```diff
@@ -2831,7 +2831,7 @@ el:^Παραχώρηση αυτοκινήτου
 es:^Compartir coche
 fi:^Autojen yhteiskäyttö
 fr:^Car Sharing
-hu:^Gépjármű megosztása
+hu:^Gépjárműmegosztás
```

Comment:
Same here: should we leave ```Gépjárműmegosztás|Gépjármű megosztás```?

Reply:
The same rule here like above (this is the so-called 6:3 rule: 6 sillables and 3 parts).